themes/komitywa/templates/base.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!simple/base.html" %}
+
+{% block head %}
+{{ super() }}
+   <link rel="stylesheet" href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/{{ CSS_FILE }}" />
```

Comment:
Actually I have checked and simple theme also link here to css file, so this line could not make any more sens right now - if we really want to use `komitywa.css` instead of `main.css` we should define `CSS_FILE` in settings and leave resposibility of adding these styles to simple style.